docs/core-blockchain-concepts/proof-of-stake/stake-validation-height.md

```diff
@@ -1,3 +1,29 @@
 # <img class="dcr-icon" src="/img/dcr-icons/TicketLive.svg" /> Stake Validation Height
 
 ---
+
+Stake Validation Height (SVH) is one of the [Blockchain Parameters](../blockchain-parameters.md).
+
+SVH is the height at which votes are required to add a new block to the top of the blockchain.
+Before SVH, Proof-of-Work miners are able to add blocks to the chain without including stakeholder votes.
+After SVH, all mined blocks must include at least 3 votes, otherwise they will not be considered
+valid by consensus rules.
+
+The SVH height is the first block that will be voted on, but will include in itself no votes.
```

Comment:
'SVH height' seems redundant, though saying height does make it read more clearly...

Reply:
Good spot. Going with just `SVH`, I would rather be grammatically correct, this is actually a bit of a pet peeve for me.

PIN Number
PAT Testing
OSS Software
LCD Display


These all annoy me :)